airflow/operators/docker_operator.py

```diff
@@ -115,6 +117,7 @@ def __init__(
             force_pull=False,
             mem_limit=None,
             network_mode=None,
+            auto_remove=False,
```

Comment:
Could you add the argument at the very end? This is to minimize the impact, and keep backward compatibility as much as possible :-)